.github/workflows/aqa.yml

```diff
@@ -1,12 +1,12 @@
-name: Run AQAvit 
+name: Run AQAvit
 
 on:
   workflow_dispatch: # Allows the job to be manually triggered
 
 env:  # Links to the JDK build under test and the native test libs
   USE_TESTENV_PROPERTIES: true
-  CUSTOMIZED_SDK_URL: https://github.com/adoptium/temurin11-binaries/releases/download/jdk-11.0.14.1%2B1/OpenJDK11U-jdk_x64_linux_hotspot_11.0.14.1_1.tar.gz
-  
+  CUSTOMIZED_SDK_URL: https://aka.ms/download-jdk/microsoft-jdk-11.0.18-linux-x64.tar.gz
```

Comment:
Should this be a parameter to the workflow itself? Something that can be set as default (which should likely be left as temurin) that can be overridden with any specific vendor's build?

_This can be a future work item if not fixed right now._